lib/Doctrine/ORM/Query/Lexer.php

```diff
@@ -117,17 +117,31 @@ public function __construct($input)
         $this->setInput($input);
     }
 
+    /**
+     * Overrides parent method in case it does not return the `u` modifier.
+     *
+     * This is required in case `doctrine/lexer` < 1.2.0 is installed and can be
+     * removed as soon as such version cannot be installed anymore, likely when
+     * dropping PHP 7.1 support.
```

Comment:
I think, we should just allow Lexer 1.2 to be installed on PHP 7.1: doctrine/lexer#58

Reply:
Lexer 1.2 is installable on PHP 7.1 now. You can bump the dependency if it helps you.

Reply:
Done.